unittest/aba-derivatives.cpp

```diff
@@ -283,7 +283,6 @@ BOOST_AUTO_TEST_CASE(test_multiple_calls)
   
   BOOST_CHECK(data1.dFdq.isApprox(data2.dFdq));
   BOOST_CHECK(data1.dFdv.isApprox(data2.dFdv));
-  BOOST_CHECK(data1.dFda.isApprox(data2.dFda));
```

Comment:
Why?

Reply:
You can a second argument to isApprox if there are some sensitivities with respect to rounding errors.

Reply:
This value is not actually computed by computeABADerivatives. It was a mistake to put it here.

Reply:
Yes, you're right.

Reply:
A copy-paste error of mine ;)